src/openseadragon.js

```diff
@@ -255,6 +255,11 @@
   * @property {Boolean} [preserveImageSizeOnResize=false]
   *     Set to true to have the image size preserved when the viewer is resized. This requires autoResize=true (default).
   *
+  * @property {Number} [minScrollDeltaMS=3]
```

Comment:
Looking through the codebase it looks like we're pretty consistently using "time" to mark such variables rather than "MS". Of course "time" is ambiguous in terms of units; on the other hand, "MS" looks like "Microsoft". At any rate, please change this to `minScrollDeltaTime` and specify that it's milliseconds in the comment.